guides/common/modules/con_available-upgrade-paths.adoc

```diff
@@ -0,0 +1,44 @@
+[id="available-upgrade-paths"]
+= Available upgrade paths
+
+Upgrading to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVersionPrevious} includes the following high-level steps:
+
+ifdef::satellite[]
+. Upgrading your {ProjectServer}s and {SmartProxyServers} to {Project} {ProjectVersionPrevious}.
+endif::[]
+ifndef::foreman-deb[]
+. Upgrading your {ProjectServer}:
+.. Upgrading your {ProjectServer} to {ProjectVersion}.
++
+ifdef::satellite[]
+.. Synchronizing the new {ProjectVersion} repositories.
+endif::[]
+. Upgrading your {SmartProxyServers} to {ProjectVersion}.
+endif::[]
+ifdef::foreman-deb[]
+. Upgrading your {ProjectServer} to {ProjectVersion}.
+. Upgrading your {SmartProxyServers} to {ProjectVersion}.
+endif::[]
+ifdef::orcharhino[]
+Because {SmartProxies} at version {ProjectVersionPrevious} will keep working with your upgraded {ProjectServer} {ProjectVersion}, you can upgrade your {SmartProxies} separately over multiple maintenance windows.
+endif::[]
+ifndef::orcharhino[]
+You can upgrade your {SmartProxies} separately over multiple maintenance windows because {SmartProxies} at version {ProjectVersionPrevious} and {ProjectVersionPrevious-Previous} will keep working with your upgraded {ProjectServer} {ProjectVersion}.
```

Comment:
Aren't the benefits also explained in the bullet points below? Perhaps leave them out here and only list them in the bullet points?

Reply:
Addressed.